authorization/role/repository/role_mapping.go

```diff
@@ -183,3 +184,15 @@ func (m *GormRoleMappingRepository) Delete(ctx context.Context, id uuid.UUID) er
 
 	return nil
 }
+
```

Comment:
Please also add more details to the Create() method comment of the role mapping repo too. So, it's clear how to create a role mapping. Some example would be great.
Maybe we should add this description to the Role Mapping object itself.

Reply:
Done